docs/api.md

```diff
@@ -140,6 +140,9 @@ Since this is a HTTP-based protocol, clients should be prepared to gracefully ha
 * Miscellaneous
     * [POST /v1/get_random_bytes](#post-v1get_random_bytes)
 
+* OAuth
+    * [POST /v1/oauth/sessions/revoke (:lock: accessToken)](#post-v1oauthsessionsrevoke)
+    * [POST /v1/oauth/keys/revoke (:lock: accessToken)](#post-v1oauthkeysrevoke)
```

Comment:
> POST /v1/session/:id/destroy

Hmm...actually, this would be the only place in the whole API where we embed a resource identifier in the URL, so maybe it's not the best idea.  `POST /v1/session/revoke` with the id in the body probably fits in better here.